### PR TITLE
Add info about tokens to Wildcard Design System

### PR DESCRIPTION
**Pull request by marisak**
Added details about three types of tokens (text, colors, and space) to the Wildcard page. Also added some links to Chromatic for reference. This is mostly what I've gleaned from going through the Wildcard file and inspecting the product; very open to corrections and suggestions!

### CODE REVIEW

**Comment by marisak** [comment]
In the medium term (between now and our next time onboarding someone?), I think some of the content in [Product team onboarding](https://handbook.sourcegraph.com/product/onboarding) could move to this page. Figma setup could potentially be its own page as well. For now, I'm opting for duplication of content.

File: content/product/design/wildcard_design_system/index.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
-# Wildcard design system
+# Wildcard Design System
```
Comment (jjinnii) [inline content/product/design/wildcard_design_system/index.md]:
```suggestion
# Wildcard design system
```

Comment (jjinnii) [reply content/product/design/wildcard_design_system/index.md]: @marisak [Sourcegraph content guidelines](https://handbook.sourcegraph.com/communication/content_guidelines/actionable_language#headings-and-subheadings) typically only capitalize the first word!

File: content/product/design/wildcard_design_system/index.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+
+## Libraries
+
+The [Wildcard Design System](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Wildcard-Design-System?node-id=891%3A0) is the primary library to use in order to have access to all of our components and styles. The [Helpers and Tooling](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/%F0%9F%9B%A0Project-Tools?node-id=72%3A20) also contains useful components like redlines and annotation.
+
+## Basic Tokens
+
+Text:
+
+- We use SF PRO for all text. [Download SF PRO](https://drive.google.com/drive/folders/1X1hwQr4lGGVn5BDe4f09q_xRqboQZpsQ) and install it to your laptop.
```
Comment (quinnkeast) [inline content/product/design/wildcard_design_system/index.md]:
```suggestion
- We use SF Pro for all text. [Download SF Pro](https://drive.google.com/drive/folders/1X1hwQr4lGGVn5BDe4f09q_xRqboQZpsQ) and install it on your system.
```

**Review by quinnkeast — APPROVED** [review]

**Review by jjinnii — APPROVED** [review]

**Review by rrhyne — APPROVED** [review]
Looks great.

**Pull request merged**